### PR TITLE
Create ui-c8y-1021-0-9-nightly-failure-data-and-control-team-device-protocol-loriot-7512.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [DM-4173](https://cumulocity.atlassian.net/browse/DM-4173)
Original PR: [7512](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7512)

[DM-4173]: https://cumulocity.atlassian.net/browse/DM-4173?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/change-logs/device-management/ui-c8y-1021-0-9-nightly-failure-data-and-control-team-device-protocol-loriot-7512.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Nightly failure dataAndControlTeam deviceProtocolLoriot (#7512)
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1021-0-9-nightly-failure-data-and-control-team-device-protocol-loriot-7512.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Fix nightly test failure for Loriot device protocol
```

File: content/change-logs/device-management/ui-c8y-1021-0-9-nightly-failure-data-and-control-team-device-protocol-loriot-7512.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-4173
+version: 1021.0.9
+---
+Nightly failure dataAndControlTeam deviceProtocolLoriot (#7512)
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1021-0-9-nightly-failure-data-and-control-team-device-protocol-loriot-7512.md:17]: The Claude AI suggested to use the following body:
```suggestion
The nightly tests for the Loriot device protocol were failing consistently. The root cause was identified and fixed by correcting the request path used in the tests. With this change, the nightly tests for the Loriot device protocol are passing again. There is no impact to any Loriot devices or functionality.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Comment by BeateRixen** [comment]
@l2p- @dmajsag I assume that we can close this PR as this change should not get a change log?

**Pull request closed without merging**